js/client.js

```diff
@@ -2741,15 +2741,27 @@ function toId() {
 		initialize: function (data) {
 			var buf = '';
 			buf = '<p>Your replay has been uploaded! It\'s available at:</p>';
-			buf += '<p><a href="https://' + Config.routes.replays + '/' + data.id + '" target="_blank" class="no-panel-intercept">https://' + Config.routes.replays + '/' + data.id + '</a></p>';
-			buf += '<p><button class="autofocus" name="close">Close</button></p>';
+			buf += '<p> <a id="replayLink" href="https://' + Config.routes.replays + '/' + data.id + '" target="_blank" class="no-panel-intercept">https://' + Config.routes.replays + '/' + data.id + '</a></p>';
+			buf += '<p><button class="autofocus" name="close">Close</button> <button name="copyReplayLink">Copy</button><p>';
 			this.$el.html(buf).css('max-width', 620);
 		},
 		clickClose: function () {
 			this.close();
 		},
 		submit: function (i) {
 			this.close();
+		},
+		copyReplayLink: function () {
+			var copyText = document.querySelector("#replayLink");
+			var dummyReplayLink = document.createElement("input");
+			// This is a hack. You can only "select" an input field. The trick is to create a short lived input element and destroy it after a copy.
+			dummyReplayLink.id = "dummyReplayLink";
+			dummyReplayLink.value = copyText.href;
+			dummyReplayLink.setAttribute("style", "absolute; left: -1000px; top: -1000px");
```

Comment:
```suggestion
			dummyReplayLink.style.position = 'absolute';
```

You still can't set `style="absolute"`, you have to set `style="position: absolute"`.

All this is academic since we remove the child immediately after appending it. There's only a single setting that would have any effect, and that's `position`, so the rest of the layout doesn't need to be deformed in certain rendering engines.